src/monodroid/jni/monodroid-glue.cc

```diff
@@ -2028,6 +2028,11 @@ MonodroidRuntime::get_java_class_name_for_TypeManager (jclass klass)
 	JNIEnv *env = osBridge.ensure_jnienv ();
 	jstring name = reinterpret_cast<jstring> (env->CallObjectMethod (klass, Class_getName));
 	const char *mutf8 = env->GetStringUTFChars (name, nullptr);
```

Comment:
Shouldn't `GetStringUTFChars` have 3 parameters?

In https://docs.oracle.com/javase/7/docs/technotes/guides/jni/spec/functions.html I see:

`const char * GetStringUTFChars(JNIEnv *env, jstring string,
jboolean *isCopy);`

Reply:
Ah, nm. I missed the `env->` part :-)